src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +690,31 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
+    ) -> bool:  # TODO: add the anomaly type once that's added to Sentry
+        """
+        Helper function to determine whether we care about an anomaly based on the
+        anomaly type and the alert rule's threshold type.
+        """
+        # need to define constants for anomaly types
+        anomaly_type = anomaly.anomaly_type
```

Comment:
Somewhere we need to handle if the user has opted into getting warning level alerts as well, I can't remember if that's in the Seer response currently or not

Reply:
I don't think it is yet, so I'll add a note to come back to it

Reply:
I'll probably have the function return an enum to handle the various cases